resources/events.hubdb.json

```diff
@@ -29,7 +29,19 @@
       "label": "Registered Attendee Count",
       "type": "NUMBER"
     },
-    { "name": "form_guid", "label": "Event Form", "type": "TEXT" }
+    { "name": "form_guid", "label": "Event Form", "type": "TEXT" },
```

Comment:
removing the form_guid column from the sample json and from any tables in-use will prevent conflicts after #34

Reply:
Working on this rn :)